lab_5_scrapper/scrapper.py

```diff
@@ -127,11 +227,29 @@ def _extract_url(self, article_bs: BeautifulSoup) -> str:
         Returns:
             str: Url from HTML
         """
+        url = ''
+        for div in article_bs.find_all("div", {"class": "title"}, limit=24):
```

Comment:
why do you have this limit?

Reply:
it was a maximum number of articles per page, but I figured out how to do it without this limit